vxquery-core/src/main/java/org/apache/vxquery/serializer/XMLSerializer.java

```diff
@@ -446,29 +525,7 @@ private void printSequence(PrintStream ps, SequencePointable seqp, String betwee
         }
     }
 
-    private void printArray(PrintStream ps, TaggedValuePointable tvp) {
-        ArrayPointable ap = pp.takeOne(ArrayPointable.class);
-        try {
-            tvp.getValue(ap);
-            int len = ap.getEntryCount();
-            ps.append('[');
-            ps.append(' ');
-            for (int i = 0; i < len; i++) {
-                ap.getEntry(i, tvp);
-                print(tvp.getByteArray(), tvp.getStartOffset(), tvp.getLength(), ps);
-                if (i != len - 1) {
-                    ps.append(',');
-                }
-                ps.append(' ');
-            }
-            ps.append(']');
-        } finally {
-            pp.giveBack(ap);
-            pp.giveBack(tvp);
-        }
-    }
-
-    private void printBase64Binary(PrintStream ps, TaggedValuePointable tvp) {
+     private void printBase64Binary(PrintStream ps, TaggedValuePointable tvp) {
```

Comment:
Try to minimize your changes to the rest of the code base unless you have a good argument as to why things should be moved. Its easier to track changes when each change do not include reformatting or moving of code.